libraries/AP_Mount/AP_Mount_Xacti.cpp

```diff
@@ -491,41 +499,41 @@ bool AP_Mount_Xacti::handle_param_get_set_response_int(AP_DroneCAN* ap_dronecan,
 {
     // display errors
     const char* err_prefix_str = "Xacti: failed to";
-    if (strcmp(name, XACTI_PARAM_SINGLESHOT) == 0) {
+    if (strcmp(name, get_param_name_str(Param::SingleShot)) == 0) {
```

Comment:
```suggestion
    if (name_is_for_parameter(name, Param::SingleShot)) {
```

Reply:
Hi @peterbarker, thanks again for the review.  I agree having this method is a nice cleanup although what we have there now is consistent with other DroneCAN drivers.

I'm happy to make the change in the future but I'd prefer if we didn't make this a blocker for this PR.